subprojects/test-runner/client/src/main/kotlin/com/avito/runner/scheduler/metrics/model/DeviceWorkerEvents.kt

```diff
@@ -0,0 +1,38 @@
+package com.avito.runner.scheduler.metrics.model
+
+import com.avito.math.Percent
+import com.avito.math.percentOf
+import java.time.Duration
+
+internal sealed class DeviceWorkerEvents {
```

Comment:
As for me, that name sounds understandable

Reply:
`DeviceWorkerEvents` -> `DeviceWorkerEvent` ?
Because this is a sealed class and describes a typed event.

Reply:
Will rename